packages/story-editor/src/components/floatingMenu/layer.js

```diff
@@ -99,9 +128,20 @@ function FloatingMenuLayer() {
       .forEach((node) => observer.observe(node, { attributes: true }));
 
     return () => observer.disconnect();
-  }, [moveable]);
+  }, [moveable, hasMenu]);
 
-  return <FloatingMenu ref={menuRef} />;
+  if (!hasMenu) {
+    return false;
+  }
+
+  return (
+    <FloatingMenu
+      ref={menuRef}
+      dismiss={dismiss}
+      hasMultiSelection={hasMultiSelection}
+      selectionTypes={selectionTypes}
```

Comment:
`hasMultiSelection` and `selectionTypes` don't seem to be used now. Are these for later?

Reply:
Yes, that was because they were needed for the next stage.